docs/tools/dotnet-Commands.md

```diff
@@ -19,9 +19,10 @@ For complete information on `dotnet`, see [.NET Core command-line interface (CLI
 - [**dotnet remove package**](/dotnet/core/tools/dotnet-remove-package): Removes a package reference from the project file.
 - [**dotnet restore**](/dotnet/core/tools/dotnet-restore?tabs=netcore2x): Restores the dependencies and tools of a project. As of NuGet 4.0, this runs the same code as `nuget restore`.
 - [**dotnet nuget locals**](/dotnet/core/tools/dotnet-nuget-locals): Lists locations of the *global-packages*, *http-cache*, and *temp* folders and clears the contents of those folders.
+- [**dotnet new nugetconfig**](/dotnet/core/tools/dotnet-new): Creates a [`nuget.config`](https://docs.microsoft.com/en-us/nuget/reference/nuget-config-file) file to configure NuGet's behavior.
 
 ## Package creation
 
-- [**dotnet pack**](/dotnet/core/tools/dotnet-pack?tabs=netcore2x): Packs the code into a NuGet package. As of NuGet 4.0, this runs the same code as `nuget pack`.
```

Comment:
I removed this sentence as I don't think customers are looking for implementation details on this page, as this page is an introduction to the `dotnet tool`. Please let me know if you'd like me to re-add this line.

Reply:
I don't see it as implementation details, rather as toolset parity. 

I think that idea should stay in one way or another. 
So don't say code, but suggest that dotnet pack and `NuGet.Build.Tasks.Pack` are equivalent.

Reply:
The bit Loic took out doesn't say that `dotnet pack` is equivilent to `NuGet.Build.Tasks.Pack`. It's claiming that `nuget.exe pack` is the same, but I believe it is not and therefore the sentence should be removed because it's factually incorrect, no?

Reply:
Yeah, after mulling over this some more I think that `NuGet.Build.Tasks.Pack is more of an advanced scenario necessary only necessary for MSBuild. I'm leaning more towards not including this note.

Reply:
and what was the outcome of the mulling?